website/docs/d/nsxt_edge_cluster.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edge_cluster"
+sidebar_current: "docs-vcd-data-source-nsxt-edge-cluster"
+description: |-
+  Provides a data source for available NSX-T Edge Clusters.
+---
+
+# vcd\_nsxt\_edge\_cluster
+
+Provides a data source for available NSX-T Edge Clusters.
+
+Supported in provider *v3.1+*
+
+~> **Note:** This resource uses new VMware Cloud Director
```

Comment:
No need for yellow note. Better blue note.

Reply:
Changed.